tasks/cockpit-tasks

```diff
@@ -113,14 +113,30 @@ function run_one_task() {
     fi
 }
 
+function run_from_queue() {
+    cd "$XDG_CACHE_HOME"/cockpit-project/bots
+    # Consuming directly from queue, commands have bots/ prepended
+    [ -e bots ] || ln -s . bots
+    if /usr/bin/timeout 12h /bin/sh -c "set -ex; ./run-queue --amqp \"$AMQP_SERVER\""; then
```

Comment:
You can run `/bin/sh -exc ./run-queue ...` to save one level of quoting and make this look nicer.